src/python/grpcio/grpc/framework/core/_transmission.py

```diff
@@ -246,7 +268,9 @@ def advance(self, initial_metadata, payload, completion, allowance):
 
   def timeout(self, timeout):
     """See _interfaces.TransmissionManager.timeout for specification."""
-    if self._transmitting:
+    if self._abort.kind is not _Abort.Kind.NOT_ABORTED:
```

Comment:
There's probably a nicer way to phrase this w/o double negatives, but nothing comes immediately to mind, so, meh.